packages/flutter/lib/src/material/navigation_drawer.dart

```diff
@@ -478,6 +490,16 @@ class _NavigationDrawerDestinationInfo extends InheritedWidget {
   /// to 1 (selected).
   final Animation<double> selectedAnimation;
 
+  /// The color of the indicator.
+  ///
+  /// This is used by destinations to override the indicator color.
+  final Color? indicatorColor;
```

Comment:
The indicatorColor and indicatorShape properties should explain how they're related to NavigationDrawerTheme and what the default values are.